docs/06_how-to-guides/20_authorization/how_to_restrict_access_to_an_action_by_user.md

```diff
@@ -76,3 +76,7 @@ void hi( name user ) {
 
 [[info | Error message is not custom]]
 | Note that this time, as well as previous method, you can not customize the yielded error message, it will be a generic authorization error message.
+
+## Summary
+
+In conclusion, the above instructions show how to how to perform authorization checks in a smart contract.
```

Comment:
to many how to's

Reply:
hnmm I thought I fixed this already
thank you!